src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -46,8 +46,9 @@ export const SubmissionHistory = () => {
 
   const cancelRunSet = async submissionId => {
     try {
-      await Ajax(signal).Cbas.runSets.cancel(submissionId)
-      notify('success', 'Abort submission request submitted successfully', { message: 'You may refresh the page to get most recent status changes.', timeout: 5000 })
+      if (isRunSetInTerminalState(submissionId.state) === false) {
```

Comment:
I think this condition would be redundant here because the user won't be able to reach here is the button is already disabled

Reply:
Yes, I meant to get rid of this, thank you for catching!